src/components/WwwFrame/EndOfYearAppealBanner/AppealBanner.vue

```diff
@@ -1,22 +1,17 @@
 <template>
-	<div class="appeal-banner-wrapper">
+	<div class="appeal-banner-wrapper" v-if="showAppeal">
 		<div class="appeal-banner">
 			<div class="row"
 				@click="toggleAccordion">
 				<div class="small-2 show-for-large"></div>
 				<div class="small-10">
-					<!-- IF REGULAR APPEAL BANNER -->
-					<!-- PUT CONDITIONAL HERE -->
-					<h2>Your donation will power impact and hope in 2019
-
-						<!-- IF MATCHED APPEAL BANNER -->
-						<!-- PUT CONDITIONAL HERE -->
-						<!-- <h2>Double or triple the impact of your donation!</h2> -->
-
+					<h2>
 						<!-- IF BONUS APPEAL BANNER -->
-						<!-- PUT CONDITIONAL HERE -->
-						<!-- <h2>Donate today and receive a bonus to lend!</h2> -->
-
+						<span v-if="appealBonusEnabled">Donate today and receive a bonus to lend!</span>
+						<!-- IF MATCHED APPEAL BANNER -->
+						<span v-else-if="appealMatchEnabled">Double or triple the impact of your donation!</span>
+						<!-- ELSE STANDARD APPEAL BANNER -->
+						<span v-else>Your donation will power impact and hope in 2019</span>
```

Comment:
It's easiest to put our Default content as the last option in the v-if cascade.

Reply:
Makes sense 👍